src/lib/get-costume-url.js

```diff
@@ -7,30 +8,29 @@ const getCostumeUrl = (function () {
     let cachedAssetId;
     let cachedUrl;
 
-    return function (assetId, vm) {
+    return function (asset) {
 
-        if (cachedAssetId === assetId) {
+        if (cachedAssetId === asset.assetId) {
             return cachedUrl;
         }
 
-        cachedAssetId = assetId;
+        cachedAssetId = asset.assetId;
 
-        const storage = vm.runtime.storage;
-        const asset = storage.get(assetId);
         // If the SVG refers to fonts, they must be inlined in order to display correctly in the img tag.
         // Avoid parsing the SVG when possible, since it's expensive.
         if (asset.assetType === storage.AssetType.ImageVector) {
-            const svgString = vm.runtime.storage.get(assetId).decodeText();
+            const svgString = asset.decodeText();
             if (svgString.match(HAS_FONT_REGEXP)) {
                 const svgRenderer = new SVGRenderer();
                 svgRenderer.loadString(svgString);
                 const svgText = svgRenderer.toString(true /* shouldInjectFonts */);
                 cachedUrl = `data:image/svg+xml;utf8,${encodeURIComponent(svgText)}`;
             } else {
-                cachedUrl = vm.runtime.storage.get(assetId).encodeDataURI();
+                cachedUrl = asset.encodeDataURI();
             }
+            cachedUrl = asset.encodeDataURI();
```

Comment:
It looks like this line is extra and also overwriting the url data from the font injection case...

Reply:
Whoops — that's a merge error.